### ISSUE TITLE
Missing file gradle.properties results in "Build failed with an exception."

### ISSUE THREAD

**Issue by mtstv** [Adobe-Consulting-Services/lazybones-aem-templates#21]
If there is no  file
D:\_Developing\acs\aem-lazybone-template\gradle.properties
I get the next error report

```
$ ./gradlew installAllTemplates

FAILURE: Build failed with an exception.

* Where:
Build file 'D:\_Developing\acs\lazybones-aem-templates\build.gradle' line: 19

* What went wrong:
A problem occurred evaluating root project 'lazybones-aem-templates'.
> Could not find property 'bintrayUsername' on root project 'lazybones-aem-templates'.

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 2.775 secs

```

**Comment by mtstv** [comment]
fixed

**Comment by justinedelson** [comment]
this is fixed in bf163c3662f9a6d1730820ebdde1ac8472f1e3fe

**Comment by mtstv** [comment]
ok
